updater.go

```diff
@@ -89,9 +90,12 @@ func (u *Updater) update(ctx Context, options UpdateOptions) (*Update, *Error) {
 		return update, promptErrPtr(err)
 	}
 	switch updateAction {
-	case UpdateActionApply, UpdateActionAuto:
-		// Continue
+	case UpdateActionApply:
+		ctx.ReportAction(UpdateActionApply, options)
+	case UpdateActionAuto:
+		ctx.ReportAction(UpdateActionAuto, options)
```

Comment:
OK. The way it's currently implemented, if you send `apply` then the auto bit is set to `false`, and if you send `auto`, the auto bit is set to `true`:

https://github.com/keybase/keybase/pull/647/files#diff-09c19ea087354ad87b44929d4a676ff1R224